php/login.php

```diff
@@ -24,20 +24,6 @@
 		}
 	}
 	
-	if (empty($username) || empty($password)) {
-		$response = [
-			'error' => true,
-			'msg' => 'One or more blank fields.'
-		];
```

Comment:
Woops idd

On Wednesday, 23 March 2016, Emil Hammarström <notifications@github.com>
wrote:

> In php/login.php
> <https://github.com/initiumSrc/EDA216_KrustyKookies/pull/11#discussion_r57217534>
> :
>
> > @@ -24,20 +24,6 @@
> >  		}
> >  	}
> >  	
> > -	if (empty($username) || empty($password)) {
> > -		$response = [
> > -			'error' => true,
> > -			'msg' => 'One or more blank fields.'
> > -		];
>
> wooops
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/initiumSrc/EDA216_KrustyKookies/pull/11/files/d5748af36c67ff52f5abb0edd3b38fe914bfe6f6#r57217534>
>